e2e/tests-dfx/wallet.bash

```diff
@@ -176,3 +176,45 @@ teardown() {
     assert_command_fail dfx wallet upgrade
     assert_match "There is no wallet defined for identity 'default' on network 'local'.  Nothing to do."
 }
+
+@test "redeem-faucet-coupon can set a new wallet and top up an existing one" {
+    dfx_new hello
+    dfx_start
+    setup_actuallylocal_shared_network
```

Comment:
The "actuallylocal" network is used for tests that are testing some functionality that uses a network that is configured as a "network provider" (see ConfigNetworkProvider, meaning it only defines a url), but that "actually" points to the local network.  So if it isn't required (and my guess is that it isn't here), it's better to use the normal local network definition.
```suggestion
```